response_operations_ui/views/messages.py

```diff
@@ -146,3 +171,18 @@ def _get_to_name(message):
     except IndexError:
         name = message.get('msg_to')[0]
     return name
+
+
+def _thread_refine(message):
+    return {
+        'body': message.get('body'),
+        'subject': message.get('subject'),
+        'username': message.get('@msg_from').get('emailAddress'),
+        # TODO use survey ref instead of survey id
+        'survey': message.get('survey'),
+        'ruref': message.get('@ru_id').get('sampleUnitRef'),
+        'business': message.get('@ru_id').get('name'),
+        'to': _get_to_name(message),
+        'sentDate': message.get('sent_date'),
+        'internal': message.get('from_internal')
+    }
```

Comment:
Some of this functionality duplicates the `_refine` method, but with some subtle differences:
- `business_name` has become `business`
- `ru_ref` is now `ruref` and is `@ru_id. business` rather than `ru_id`

We aim to make these consistent, possibly sharing the common mappings.

Also, `sentDate` should be `sent_date`.